setup.cfg

```diff
@@ -2,8 +2,74 @@
 universal = 1
 
 [metadata]
-description-file = README.md
+name = scapy
+version = attr: scapy.VERSION
+author = Philippe BIONDI
+author_email = phil(at)secdev.org
+maintainer = Pierre LALET, Gabriel POTTER, Guillaume VALADON
+license = GPL-2.0-only
 license_file = LICENSE
+description = Scapy: interactive packet manipulation tool
+keywords = network
+url = https://scapy.net
+download_url = https://github.com/secdev/scapy/tarball/master
+long_description = file: README.md
```

Comment:
Isn't this incompatible with the fact that we have a `setup(long_description=...)` in `setup.py` still?